index.js

```diff
@@ -2,20 +2,61 @@ const config = require('./server/config')
 const { log } = require('./server/lib/util/logger')
 const PubSub = require('./server/lib/pubsubNotifiers/pubsubNotifier')
 
-let { pubsubConfig, postgresConfig } = config
-let { port } = config.server
+class DataSyncService {
```

Comment:
I do not want be another "clean code" experts, but if we extract this class to separate file then we could avoid `require.main === module`. It will be just easy and nice